csharp/src/Google.Protobuf.Test/JsonParserTest.cs

```diff
@@ -549,9 +549,12 @@ public void NumberToDouble_Valid(string jsonValue, double expectedParsedValue)
         }
 
         [Test]
+#if !NETCOREAPP3_0
+        // .NET Core 3.0 returns infinity
```

Comment:
is it a problem in .NET core 3 or wrong testcases? Maybe we could come up with better test cases that test the same kind of stuff and work on all platforms.

Reply:
I don't think it is a problem in .NET Core 3.0. Its likely an intentional change in behavior. Would you like me to look into it more?